python_modules/libraries/dagster-tableau/dagster_tableau/translator.py

```diff
@@ -15,6 +15,11 @@ def _clean_asset_name(name: str) -> str:
     return re.sub(r"[^a-z0-9A-Z.]+", "_", name).lower()
 
 
+@record
+class StartWorkbookRefreshRequest:
```

Comment:
nit: docstring, since this is user facing

Reply:
Class removed in [3c010c6](https://github.com/dagster-io/dagster/pull/24862/commits/3c010c64b1b5b07b3724aacbd4b833cc893152f6)